internal/teststep/file.go

```diff
@@ -84,11 +85,24 @@ func (c configurationFile) Write(ctx context.Context, dest string) error {
 		configFile = filepath.Join(pwd, configFile)
 	}
 
-	err := copyFile(configFile, dest)
-
+	destPath, err := copyFile(configFile, dest)
 	if err != nil {
 		return err
 	}
 
+	if len(c.appendedConfig) > 0 {
+		err := appendToFile(destPath, c.appendedConfig)
```

Comment:
Wouldn't `destPath` always be equal to `dest` unless there is an error? (in which case we should have returned prior to this I think)

Reply:
Does https://github.com/hashicorp/terraform-plugin-testing/pull/494#discussion_r2077766538 address this?

Reply:
Yes! Thanks for pointing that 👍🏻